_security-analytics/api-tools/correlation-eng.md

```diff
@@ -0,0 +1,224 @@
+      "category": "network"
+    },
+    {
+      "index": "windows",
+      "query": "winlog.event_data.SubjectDomainName:NTAUTHORI*",
+      "category": "windows"
+    },
+    {
+      "index": "ad_logs",
+      "query": "ResultType:50126",
+      "category": "ad_ldap"
+    },
+    {
+      "index": "app_logs",
+      "query": "endpoint:/customer_records.txt",
+      "category": "others_application"
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "_id": "DxKEUIkBpIjg64IK4nXg",
+  "_version": 1,
+  "rule": {
+    "name": null,
+    "correlate": [
+      {
+        "index": "vpc_flow",
+        "query": "dstaddr:4.5.6.7 or dstaddr:4.5.6.6",
+        "category": "network"
+      },
+      {
+        "index": "windows",
+        "query": "winlog.event_data.SubjectDomainName:NTAUTHORI*",
+        "category": "windows"
+      },
+      {
+        "index": "ad_logs",
+        "query": "ResultType:50126",
+        "category": "ad_ldap"
+      },
+      {
+        "index": "app_logs",
+        "query": "endpoint:/customer_records.txt",
+        "category": "others_application"
+      }
+    ]
+  }
+}
+```
+
+### Response fields
+
+| Field | Type | Description |
+| :--- | :--- |:--- |
+| `_id` | String | The new rule's ID. |
+
+
+## List all findings and their correlations within a time window
+
+Provides a list of all findings and their correlations within a specified window of time.
+
+```json
+GET /_plugins/_security_analytics/correlations?start_timestamp=<start time in milliseconds>&end_timestamp=<end time in milliseconds>
+```
+
+### Query parameters
+
+| Parameter | Type | Description |
+| :--- | :--- |:--- |
+| `start_timestamp` | Number | Start time for the time window in milliseconds. |
+| `end_timestamp` | Number | End time for the time window in milliseconds. |
+
+#### Example request
+
+```json
+GET /_plugins/_security_analytics/correlations?start_timestamp=1689289210000&end_timestamp=1689300010000
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "findings": [
+    {
+      "finding1": "931de5f0-a276-45d5-9cdb-83e1045a3630",
+      "logType1": "network",
+      "finding2": "1e6f6a12-83f1-4a38-9bb8-648f196859cc",
+      "logType2": "test_windows",
+      "rules": [
+        "nqI2TokBgL5wWFPZ6Gfu"
+      ]
+    }
+  ]
+}
+```
+
+### Response fields
+
+| Field | Type | Description |
+| :--- | :--- |:--- |
+| `finding1` | String | The ID for a first finding in the correlation. |
+| `logType1` | String | The log type associated with the first finding. |
+| `finding2` | String | The ID for a second finding in the correlation. |
+| `logType2` | String | The log type associated with the second finding. |
+| `rules` | Array | A list of correlation rule IDs associated with the correlated findings. |
+
+
+## List correlations for a finding belonging to a log type
+
+Used to list correlations for specific findings and the log types associated with them.
```

Comment:
Please rephrase as a complete sentence naming the noun (what is used to do this?).